docs/management/api/blocklist-api.asciidoc

```diff
@@ -0,0 +1,618 @@
+      "updated_by": "elastic"
+    }
+  ],
+  "page": 1,
+  "per_page": 10,
+  "total": 1
+}
+--------------------------------------------------
+
+include::_exceptions-api-reusable-content.asciidoc[tag=endpoint-exceptions-value-types-info-section]
+
+[#exception-{endpoint-artifact-api-doc-name}-item-entry-schema]
+===== `entry` object schema
+
+At least 3 conditions can be defined on an endpoint blocklist entry. Duplicates are not allowed. The following entries are supported by {endpoint-artifact-name}:
+
+====== File hashes
+
+File hashes are supported by all xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[OS types]. A file hash entry has the following structure:
+
+* `field` : The `file.hash.md5`, `file.hash.sha1` or `file.hash.sha256` values are supported.
+* `value` : An array of hash values associated with the `field`.
+* `type` : Must be `match_any`.
+* `operator` : Must be `included`.
+
+[source,json]
+-----------------------------
+{
+    "field": "file.hash.md5",
+    "value": [
+        "741462ab431a22233C787BAAB9B653C7"
+    ],
+    "type": "match_any",
+    "operator": "included"
+}
+-----------------------------
+
+
+====== Process file path
+
+Process file paths are supported by all xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[OS types]. A file path entry has the following structure:
+
+* `field` : Must be `file.path`.
+* `value` : An array of file paths to match on.
+* `type` : Must be `match_any`.
+* `operator` : Must be `included`.
+
+[source,json]
+-----------------------------
+{
+  "field": "file.path",
+  "value": "c:/path/to/file.exe",
+  "type": "match_any",
+  "operator": "included"
+}
+-----------------------------
+
+
+====== Process signature
+
+Process signature is supported for Windows xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[OS type] only. A signature entry has the following structure:
+
+* `field` : Must be set to `process.Ext.code_signature`.
+* `type` : Must be set to `nested`.
+* `entries`: An array with 1 `entry` item.
+* `entries[0]` : The entry defining the signature to be matched upon:
+** `field`: Must be set to `subject_name`.
+** `value` : An array of signature names to match on.
+** `type` : Must be set to `match_any`.
+** `operator` : Must be set to `included`.
+
+[source,json]
+-----------------------------
+{
+    "field": "file.Ext.code_signature",
+    "entries": [
+        {
+            "field": "subject_name",
+            "value": [
+                "test1",
+                "test2"
+            ],
+            "type": "match_any",
+            "operator": "included"
+        }
+    ],
+    "type": "nested"
+}
+-----------------------------
+
+
+====== Examples for a Windows {endpoint-artifact-name-single}. 
+
+Only one entry of `file.path` or `file.Ext.code_signature` is allowed per blocklist item. The `file.hash.*`entry is also supported and accepts up to three values. The allowed hash types for `file.hash.*` are `md5`, `sha1`, or `sha256`.
```

Comment:
```suggestion
Only one entry of `file.path` or `file.Ext.code_signature` is allowed per blocklist item. The `file.hash.*` entry is also supported and accepts up to three values. The allowed hash types for `file.hash.*` are `md5`, `sha1`, or `sha256`.
```

Reply:
Just missing a space. Otherwise, looks good - thanks @kevinlog!

Reply:
@nastasha-solomon fixed the typo.